eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -36,6 +36,7 @@ stages:
       dependsOn: []
       jobs:
         - template: /eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+          # Code coverage is enabled by default for live tests
```

Comment:
It is probably still worth passing the parameter explicitly here so that it is clear that the values are supposed to remain connected.